### PR TITLE
added an Unstructured.io test

### PR DESCRIPTION
**Pull request by epinzur**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -21,6 +21,7 @@ pass_env =
     LANGCHAIN_PROJECT
     LLAMA_CLOUD_API_KEY
     NVIDIA_API_KEY
```
Comment (nicoloboschi) [inline tox.ini:23]: this is for the notebooks, guess you can remove it

File: ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    context = handler.before_test()
+    yield context
+    handler.after_test()
+
+
+@pytest.mark.parametrize("vector_store", ["cassandra", "astra_db"])
+@pytest.mark.parametrize("unstructured_mode", ["single", "elements"])
+def test_unstructured(vector_store, unstructured_mode, request):
+    set_current_test_info(
+        "lang_chain::unstructured",
```
Comment (nicoloboschi) [inline ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py:46]:
```suggestion
        "langchain::unstructured-api",
```

File: ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+def cassandra():
+    handler = get_vector_store_handler(VectorStoreImplementation.CASSANDRA)
+    context = handler.before_test()
+    yield context
+    handler.after_test()
+
+
+@pytest.mark.parametrize("vector_store", ["cassandra", "astra_db"])
+@pytest.mark.parametrize("unstructured_mode", ["single", "elements"])
+def test_unstructured(vector_store, unstructured_mode, request):
```
Comment (nicoloboschi) [inline ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py:44]:
```suggestion
def test_unstructured_api(vector_store, unstructured_mode, request):
```

File: ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from langchain_community.document_loaders import UnstructuredAPIFileLoader
```
Comment (nicoloboschi) [inline ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py:3]: I believe we should have a pinned version of the client in the pyproject.toml of ragstack. 
We should find the current range configured in langchain-community and pick a version

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (epinzur) [reply tox.ini:23]: I'll be adding an example notebook in my next PR. So I'll leave for now if that is okay.

**Review by nicoloboschi — APPROVED** [review]

**Pull request merged**